wp-content/themes/midwestenergynews/inc/post-tags.php

```diff
@@ -5,14 +5,27 @@
  *
  * Note that this completely ignores the Largo_Byline class.
  *
+ * This MUST be compatible with Largo's implementation: https://github.com/INN/largo/blob/v0.6.4/inc/post-tags.php#L123
+ *
+ * @param Boolean $echo Echo the string or return it (default: echo)
+ * @param Boolean $exclude_date Whether to exclude the date from byline (default: false)
+ * @param WP_Post|Integer $post The post object or ID to get the byline for. Defaults to current post.
+ * @return String Byline as formatted html
  */
 function largo_byline( $echo = true, $exclude_date = false, $post_id = null ) {
```

Comment:
This might need to be modified depending on the answer to the question on `content-single-classic.php`.